standards/collections.md

```diff
@@ -258,7 +259,7 @@ RESPONSE
 
 Cursor-based Pagination works extremely well when:
 
-- Offset-based pagination does not scale as you get further down the index calculating the offset on large sets of data.
+- All collection-based endpoints **SHOULD** support cursor-based over offset-based pagination unless technology or performance makes offset-based pagination more advantageous.
```

Comment:
I'm not sure this needed to change? Maybe just adding a new item that indicates cursor-based pagination works well as a default to prevent paginiation contracts from changing as the API matures.